include/dlaf/common/index2d.h

```diff
@@ -261,6 +261,16 @@ Coords2DType transposed(Coords2DType coords) {
   return coords;
 }
 
+template <class IndexT, class TAG>
+Index2D<IndexT, TAG> indexFromOrigin(const Size2D<IndexT, TAG>& size) noexcept {
+  return {size.rows(), size.cols()};
+}
+
+template <class IndexT, class TAG>
+Size2D<IndexT, TAG> sizeFromOrigin(const Index2D<IndexT, TAG>& size) noexcept {
+  return {size.row(), size.col()};
+}
```

Comment:
Aren't these two swapped? I would have associated `rows()` to `Size2D` while `row()` to `Index2D`.